docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -325,3 +325,44 @@ This issue might occur when you use a Zowe version of 1.12.0 or later. To resolv
 ```ZWED_node_https_certificateAuthorities="/path/to/zowe/keystore/local_ca/localca.cer-ebcdic","/path/to/carootcert.pem","/path/to/caintermediatecert.pem"```
  
 Recycle your Zowe server. You should be able to log in to the Zowe Desktop successfully now.
+
+### Browser unable to connect with a CIPHER error
+
+**Symptom:**
+
+When connecting to the API Mediation Layer the web browser throws an error that the site is unable to provide a secure connection because of an error with ciphers.  
+
+The error shown varies depending on the browser, e.g.
+
+For Google Chrome
```

Comment:
In Github's formatter, the text 
```For Google Chrome``` and ```For Mozilla Firefox``` appear between the image files; it should be both-before or both-after the images.